app/Http/Controllers/UserController.php

```diff
@@ -188,6 +189,10 @@ public static function getProfilePage($username) {
 
     }
 
+    /** 
```

Comment:
a "delete all notifications" is missing from the `DestroyUser`-method

Reply:
Fixed with b9c220e